src/core/config/from_document.rs

```diff
@@ -449,6 +458,7 @@ trait Fieldlike {
     fn type_of(&self) -> &Type;
     fn description(&self) -> &Option<Positioned<String>>;
     fn directives(&self) -> &[Positioned<ConstDirective>];
+    fn default_value(&self) -> Option<ConstValue>;
```

Comment:
We don't need this to be implemented via FieldLike

Reply:
removed it